packages/flutter/lib/src/animation/tween.dart

```diff
@@ -82,9 +82,10 @@ abstract class Animatable<T> {
   }
 
   /// Returns a new [Animatable] whose value is determined by first evaluating
-  /// the given parent and then evaluating this object.
+  /// the given parent and then evaluating this object on the result.
```

Comment:
```suggestion
  /// the given parent and then evaluating this object based on the result.
```

Reply:
The intended meaning of "evaluate" here is function evaluation, which isn't what "based on" would mean to me. But this seems like a sign this can be made clearer. Here's a version tweaked and made more explicit:

```suggestion
  /// the given parent and then evaluating this object at the result.
  ///
  /// This method represents function composition on [transform]:
  /// the [transform] method of the returned [Animatable] is the result of
  /// composing this object's [transform] method with
  /// the given parent's [transform] method.
```